src/jasmine-tests/bundle-url-rewrite-spec.js

```diff
@@ -0,0 +1,270 @@
+            givenCssFileText('url(fonts/a-font.otf#foobar)');
+
+            versionUrls();
+
+            verifyOutputTextIs("url('combined/version__18458d2016656fdb823ed3ef01b8f8da__/fonts/a-font.otf#foobar')");
+
+        });
+
+        it("Image urls with double quotes are parsed correctly.", function () {
+
+            givenFile('img/an-image.jpg', fileContents1);
+            givenCssFileText('.s { background: url("img/an-image.jpg") center no-repeat; }');
+
+            versionUrls();
+
+            verifyOutputTextIs(".s { background: url('combined/version__18458d2016656fdb823ed3ef01b8f8da__/img/an-image.jpg') center no-repeat; }");
+        });
+
+        it("Image urls with no quotes are parsed correctly.", function () {
+
+            givenFile('img/another-image.jpg', fileContents1);
+            givenCssFileText('.s { background: url(img/another-image.jpg) center no-repeat; }');
+
+            versionUrls();
+
+            verifyOutputTextIs(".s { background: url('combined/version__18458d2016656fdb823ed3ef01b8f8da__/img/another-image.jpg') center no-repeat; }");
+        });
+
+        it("Extra slashes are not added if one already exists.", function () {
+
+            givenFile('/img/another-image.jpg', fileContents1);
+            givenCssFileText('.s { background: url(/img/another-image.jpg) center no-repeat; }');
+
+            versionUrls();
+
+            verifyOutputTextIs(".s { background: url('combined/version__18458d2016656fdb823ed3ef01b8f8da__/img/another-image.jpg') center no-repeat; }");
+        });
+
+        it('Simple font faces are parsed correctly.', function() {
+
+            givenFile('/fonts/a-font/a-font.eot', fileContents1);
+            givenFile('/fonts/a-font/a-font.woff', fileContents2);
+            givenCssFileText('@font-face{font-family:a-font;src:url(/fonts/a-font/a-font.eot);src:url(/fonts/a-font/a-font.woff);}');
+
+            versionUrls();
+
+            verifyOutputTextIs('@font-face{font-family:a-font;src:url(\'combined/version__18458d2016656fdb823ed3ef01b8f8da__/fonts/a-font/a-font.eot\');src:url(\'combined/version__570e1018fd20af9e8e22c860a43d0ac3__/fonts/a-font/a-font.woff\');}');
+
+        });
+
+        it('Complex font faces are parsed correctly.', function() {
+
+            givenFile('/fonts/a-font/a-font.eot', fileContents1);
+            givenFile('/fonts/a-font/a-font.woff', fileContents2);
+            givenFile('/fonts/a-font/a-font.ttf', fileContents3);
+            givenFile('/fonts/a-font/a-font.svg', fileContents4);
+            givenCssFileText("@font-face{font-family:a-font;src:url(/fonts/a-font/a-font.eot);"
+                + "src:url(/fonts/a-font/a-font.eot?#iefix) format('embedded-opentype'),"
+                + "url(/fonts/a-font/a-font.woff) format('woff'),"
+                + "url(/fonts/a-font/a-font.ttf) format('truetype'),"
+                + "url(/fonts/a-font/a-font.svg#a-font) format('svg');"
+                + "font-weight:400;font-style:normal}");

```

Comment:
@ashley-casey-zocdoc This test verifies that font-faces, like the ones you added to styleguide typography, get rewritten correctly.